candies/cli/cli.py

```diff
@@ -44,9 +44,16 @@ def command(self, *args, **kwargs):
         Consider looking at the documentation of the `Command.command` method
         for more details.
         """
-
         return self.main.command(*args, **kwargs)
 
+    def help(self, *args, **kwargs):
+        """Defines a function to be called for `--help`.
+
+        Consider looking at the documentation of the `Command.command` method
```

Comment:
I feel like `Command.command` should be `candies.cli.command.help`.

Reply:
Yes.